<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>76</width>
         <height>76</height>
        </rect>
       </property>
       <widget class="QLabel" name="imageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="printAct"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="normalSizeAct"/>
    <addaction name="separator"/>
    <addaction name="fitToWindowAct"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="aboutQtAct"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAct">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="printAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Exit...</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="normalSizeAct">
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="toolTip">
    <string>Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
   <property name="toolTip">
    <string>Fit to Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="aboutQtAct">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
